product_configurator_wizard/wizard/product_configurator.py

```diff
@@ -783,7 +786,8 @@ def action_config_done(self):
         if self.product_id:
             self.product_id.write({
                 'attribute_value_ids': [(6, 0, self.value_ids.ids)],
-                'value_custom_ids': [(6, 0, custom_vals)]
+                'value_custom_ids': map(lambda cv: (2, cv), self.product_id.value_custom_ids.ids) + \
+                                        self.product_id.product_tmpl_id.encode_custom_values(custom_vals),
```

Comment:
So, not sure how you would like me to re-structure this, and can you tell me what senarios might generate a remove if not needed.  Happy to re-structure this according to your preferences.